pastastore/store.py

```diff
@@ -1150,13 +1162,19 @@ def get_stressmodel(
 
         # special for WellModels
         if stressmodel._name == "WellModel":
-            names = [s.squeeze().name for s in stresses["stress"]]
+            if isinstance(stresses["stress"], list):
+                names = [s.squeeze().name for s in stresses["stress"]]
+            else:
+                names = [stresses["stress"].squeeze().name]
+                stresses["stress"] = [stresses["stress"]]  # ugly fix for WellModel
```

Comment:
not sure what happens if the stress has no name?

Reply:
Good question... can that happen? Or does pastastore guarantee the series gets a name...

Reply:
Not sure haha

Reply:
I believe it does.